arm_control:
  arm_name: "manipulator"
